### PR TITLE
Add PCB Utopia88

### PR DESCRIPTION
**Pull request by Kelwin0313**
Add PCB Utopia88

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Why have you opened yet another PR?

**Comment by zvecr** [comment]
Please stop! It makes the review process much harder than it needs to be. Repeated closing and reopening as you have done many many times is likely to get you blocked from submitting PRs.

**Comment by Kelwin0313** [comment]
@zvecr I will keep everything in this PR. I understand my client has submitted like 4PRs before hiring me to deal with the PR. Sorry for that.

File: keyboards/kelwin/utopia88/info.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                {"matrix":[5,13], "x":12.5, "y":5.25}, 
+                {"matrix":[5,14], "x":13.5, "y":5.25, "w":1.5}, 
+                {"matrix":[5,15], "x":15.25, "y":5.25}, 
+                {"matrix":[5,16], "x":16.25, "y":5.25}, 
+                {"matrix":[5,17], "x":17.25, "y":5.25}
+            ]
+        }
+    }
+}
+
```
Comment (dunk2k) [inline keyboards/kelwin/utopia88/info.json:131]:
```suggestion

```

File: keyboards/kelwin/utopia88/info.json
Comment (dunk2k) [inline keyboards/kelwin/utopia88/info.json:1]: please run, qmk CLI command, `qmk format-json` against this file

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**